README.md

```diff
@@ -91,62 +92,57 @@ Smart API is a set of REST-like APIs that expose many capabilities required to b
+            orderParams.setTradingSymbol("ITC-EQ");
+            orderParams.setProductType(Constants.PRODUCT_INTRADAY);
+            orderParams.setDuration(Constants.DURATION_DAY);
+            orderParams.setTransactionType(Constants.TRANSACTION_TYPE_BUY);
+            orderParams.setPrice(122.2);
+            orderParams.setSquareOff("0");
+            orderParams.setStopLoss("0");
 
 		Order order = smartConnect.placeOrder(orderParams, Constants.VARIETY_REGULAR);
 	}
 
 	/** Modify order. */
 	public void modifyOrder(SmartConnect smartConnect) throws SmartAPIException, IOException {
 		// Order modify request will return order model which will contain only
-
-		OrderParams orderParams = new OrderParams();
-		orderParams.quantity = 1;
-		orderParams.ordertype = Constants.ORDER_TYPE_LIMIT;
-		orderParams.tradingsymbol = "ASHOKLEY";
-		orderParams.symboltoken = "3045";
-		orderParams.producttype = Constants.PRODUCT_DELIVERY;
-		orderParams.exchange = Constants.EXCHANGE_NSE;
-		orderParams.duration = Constants.VALIDITY_DAY;
-		orderParams.price = 122.2;
-
-		String orderId = "201216000755110";
+            
+            orderParams.setQuantity(324);
+            orderParams.setOrderType(Constants.ORDER_TYPE_STOPLOSS_LIMIT);
+            orderParams.setTradingSymbol("ITC-EQ");
+            orderParams.setSymbolToken("1660");
+            orderParams.setProductType(Constants.PRODUCT_INTRADAY);
+            orderParams.setExchange(Constants.EXCHANGE_NSE);
+            orderParams.setDuration(Constants.DURATION_DAY);
+            orderParams.setPrice(122.2);
+            String orderId = "201216000755110";
 		Order order = smartConnect.modifyOrder(orderId, orderParams, Constants.VARIETY_REGULAR);
 	}
-
-	/** Cancel an order */
+    
 	public void cancelOrder(SmartConnect smartConnect) throws SmartAPIException, IOException {
-		// Order modify request will return order model which will contain only
-		// order_id.
-		// Cancel order will return order model which will only have orderId.
 		Order order = smartConnect.cancelOrder("201009000000015", Constants.VARIETY_REGULAR);
 	}
 
 	/** Get order details */
-	public void getOrder(SmartConnect smartConnect) throws SmartAPIException, IOException {
-		List<Order> orders = smartConnect.getOrderHistory(smartConnect.getUserId());
-		for (int i = 0; i < orders.size(); i++) {
-			System.out.println(orders.get(i).orderId + " " + orders.get(i).status);
-		}
-	}
-
-	/**
-	 * Get last price for multiple instruments at once. USers can either pass
-	 * exchange with tradingsymbol or instrument token only. For example {NSE:NIFTY
-	 * 50, BSE:SENSEX} or {256265, 265}
-	 */
+    public void getOrder(SmartConnect smartConnect) throws SmartAPIException, IOException {
+        List<Order> orders = smartConnect.getOrderHistory(smartConnect.getUserId());
+            for (Order order : orders) {
+            logger.info(order.orderId + " " + order.status);
```

Comment:
We should use logger.info("{}  {}", order.orderId, order.status)

Reply:
have replaced String concatenation with place holder
  log.info("{} {}", order.orderId, order.status);